core/frontend/src/utils/api.ts

```diff
@@ -6,8 +6,13 @@ const backend_offline_error = new Error('Backend is offline')
 backend_offline_error.name = 'BackendOffline'
 export { backend_offline_error }
 
-const axios_backend_instance: AxiosInstance = axios.create()
+export const isBackendOffline = (error: any): boolean => {
```

Comment:
I just woke up... but is this not causing an infinite recursion?

Reply:
wrong global replace!